packages/shell-api/src/collection.ts

```diff
@@ -1561,6 +1561,7 @@ export default class Collection extends ShellApiClass {
         totalValue[`Shard ${shardStats.shardId}`] = [
           `${estDataPercent} % data`,
           `${estDocPercent} % docs in cluster`,
+          `${estDocPercent} % docs in cluster`,
```

Comment:
Is this intentional?

Reply:
Nope!! Time for me to stop working today :)